src/aind_data_transfer/transformations/converters.py

```diff
@@ -76,14 +97,16 @@ def log_to_acq_json(log_dict: dict) -> Acquisition:
                        local_storage_directory=local_storage_directory,
                        external_storage_directory=external_storage_directory)
 
-def acq_json_to_xml(acq_obj: Acquisition, s3_path: str, zarr: bool = True, condition: str = "") -> ET.ElementTree:
+
+
+def acq_json_to_xml(acq_obj: Acquisition, log_dict: dict, data_loc: str, zarr: bool = True, condition: str = "") -> ET.ElementTree:
```

Comment:
Add documentation for 'log_dict' and columns that can be filtered by for 'condition'.

Reply:
Documentation added to various functions which should make it a little more clear, if no less complicated.